h/static/styles/partials-v2/_lozenge.scss

```diff
@@ -0,0 +1,23 @@
+.lozenge {
+  display: flex;
+  background-color: $grey-3;
+  color: $grey-6;
+  border-radius: 2px;
+  margin-left: 10px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.lozenge-content {
```

Comment:
Element styles should use `component__element` naming rather than `component-element`, as per the [naming rules](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#naming) in our CSS style guide.